content/applications/finance/payment_acquirers/authorize.rst

```diff
@@ -115,6 +115,91 @@ order is canceled, you can click on *Void Transaction* to unlock the funds from
+:guilabel:`Expanded Fields with CAVV Response/Comma Separated`, enable :guilabel:`Include Column
+Headings`, and finally click :guilabel:`Submit`. Open the text file, select :guilabel:`All`, copy
+the data and paste it into the :guilabel:`Report 2 Download` sheet of the Excel file.
+
+Transit lines are automatically filled-in and updated in the :guilabel:`transit for report 1` and
+:guilabel:`transit for report 2` sheets of the Excel file. Make sure all entries are present, and if
+not, copy the **formula** from previous filled-in lines of :guilabel:`transit for report 1` or
+:guilabel:`2` and paste it into the empty lines.
+
+.. important::
+   To get the same closing balance, do *not* remove any line from the Excel sheets.
+
+Import into Odoo
+----------------
+
+To import the data into Odoo, go to the Excel file, copy the data from the :guilabel:`transit
+report 2` sheet and **paste special** only the **values** in the :guilabel:`Odoo Import to CSV`
+sheet. Then, look for *blue*-highlighted cells in the :guilabel:`Odoo Import to CSV` sheet. These
+are **chargeback** entries without any **reference** number. To fix this, go to
+:menuselection:`Authorize.Net --> Account --> Statements --> eCheck.Net Settlement Statement`, look
+for :guilabel:`Charge Transaction/Chargeback`, and click it. Copy the **invoice description**, paste
+it into the :guilabel:`Label` cell of the :guilabel:`Odoo Import to CSV` sheet, and add
+"**Chargeback /**" before the description. If you have multiple invoices, add a line into the Excel
+sheet for each invoice and copy/paste the description into each respective :guilabel:`Label` line.
+
+.. note::
+   For **chargeback/returns** combined in the payouts, you need to create a new Excel line for each
+   invoice.
+
+.. example::
+   .. image:: authorize/authorize-chargeback-desc.png
+      :align: center
+      :alt: Charge back description
+
+Next, remove **zero transaction** and **void transaction** line items, and change the **format** of
+the :guilabel:`Amount` column in the :guilabel:`Odoo Import to CSV` sheet to *Number*. Go back to
+:menuselection:`eCheck.Net Settlement Statement --> Search for a Transaction` and search again for
+the previously used batch settlements dates. Verify that the batch settlement dates on
+**eCheck.Net** match the related payments' dates found in the :guilabel:`Date` column of the
+:guilabel:`Odoo Import to CSV`. If it does not match, replace the date with the one from
+**eCheck.Net**. Sort the column by *date*, and make sure the format is `MM/DD/YYYY`. Finally, copy
+the data (column headings included) of the :guilabel:`Odoo Import to CSV` sheet, paste into a new
+Excel file of your choice, and save it as .CSV format.
+
+Open your Accounting app, go to :menuselection:`Configuration --> Journals`, tick the
+:guilabel:`Authorize.Net` box, and click :menuselection:`Favorites --> Import records`. Select the
+file saved as .CSV, and upload it into Odoo.
```

Comment:
Should we say "Go to :menuselection:`Accounting --> Configuration --> Journals` ?

There is a small step missing here: Click the :guilabel:`Load file` button.
and I would say Click the :guilabel:`Load file` button, then select your CSV file.

Reply:
I want to make it clear we are no longer using the Excel sheet but the accounting app, so I'll leave the menuselection as such